CMakeLists.txt

```diff
@@ -338,23 +332,25 @@ find_package(Threads)
 
 if(NOT CMAKE_BUILD_TYPE)
 	set(CMAKE_BUILD_TYPE "Release" CACHE STRING
-		"Build type (Release/Debug/RelWithDebInfo/MinSizeRe)" FORCE)
+		"Build type (Release/Debug/RelWithDebInfo/MinSizeRel)" FORCE)
 endif(NOT CMAKE_BUILD_TYPE)
 
 
-if(CMAKE_BUILD_TYPE STREQUAL Debug)
-	add_definitions(-D_DEBUG -ggdb)
+if(CMAKE_BUILD_TYPE STREQUAL "Debug")
+	add_definitions(-D_DEBUG)
+	check_and_add_flag(GGDB -ggdb)
+
 	set(wxWidgets_USE_DEBUG ON CACHE BOOL "Use wxWidgets Debugging")
 
 	option(ENABLE_GPROF "Enable gprof profiling (must be using Debug build)" OFF)
 	if(ENABLE_GPROF)
-		add_definitions(-pg)
+		check_and_add_flag(HAVE_PG -pg)
```

Comment:
If this can't be added, but `ENABLE_GPROF` is set by the user, it should be a hard error.